src/client/ui/panels/HierarchyPanelContainer.js

```diff
@@ -130,15 +130,17 @@ class HierarchyPanelContainer extends Component {
 
   rebuildNodeHierarchy = () => {
     const handler = this.props.editor.scene.userData._conflictHandler;
-    const list = handler.checkResolvedMissinRoot(this.props.editor.scene);
-    if (list.length > 0) {
-      list.forEach(resolvedMissingRoot => {
-        this.props.editor.removeObject(resolvedMissingRoot);
-      });
+    if (handler) {
```

Comment:
Hmm, I guess we're doing this because new scenes don't have conflictHandlers? Maybe they should. 
Doesn't have to be fixed in this PR, but this kind of `if` statement (without an `else`) could lead to some surprising bugs. Maybe just keep a mental note to come back to this.